dbc_extract3/dbc/generator.py

```diff
@@ -1297,6 +1297,8 @@ class SpellDataGenerator(DataGenerator):
          355829, 356042, 356043, 356046,
          # Banshee's Blight (Sylvanas Dagger) damage spell
          358126,
+         # Exsanguinated (Shard of Bek)
+         356372
```

Comment:
I'm not sure if this is the right place to put this. Exsanguinated (the debuff the target gets) wasn't in the spell data so I've whitelisted it and regenerated spell data and it was working after that.

I didn't push regenerated spell data since I wasn't sure if it belongs in this PR. I can push it if needed.

Reply:
Yes, this is the correct place for this. The spell data will need to be updated for the implementation to work, so you can go ahead and push it.

Reply:
Spell data is now up